orange-datamodel/pom.xml

```diff
@@ -42,5 +42,9 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
orange-datamodel should not have a dependency on hmf-common.

Reply:
The dependency was added because PurpleVariant contained a list of `VariantTranscriptImpact`'s (which is a class of `hmf-common`).

However, I made some changes that added the `PurpleVariantTranscriptImpact` class and the `VariantTranscriptImpact` instances are mapped to this class. Hence the dependency was can be removed entirely.